src/models/DateInfo.js

```diff
@@ -0,0 +1,82 @@
+export default class DateInfo {
```

Comment:
I won't review this file. Use another api resource and rethink what you will show as forecast